### DISCUSSION TITLE
blog/cloud-and-devops-trend-2022/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#289] [Announcements]
# blog/cloud-and-devops-trend-2022/

近日 InfoQ 发布了 DevOps and Cloud InfoQ Trends Report – June 2022，因为报告中所覆盖的技术领域过于宽泛，本文仅仅是对这篇报告的一点个人解读。

https://jimmysong.io/blog/cloud-and-devops-trend-2022/

**Comment by rootsongjc** [comment]
平台工程团队被列入到了创新者象限，似乎是乱入。因为这个象限中的其他都是技术，这个更像是与团队管理有关。
